docs/topics/multiplatform/multiplatform-build-native-binaries.md

```diff
@@ -493,3 +493,17 @@ for publishing to private or public podspec repositories.
 > of Kotlin. Doing so might lead to conflicts in the end-users' projects.
 >
 {type="warning"}
+
+## Customize the Info.plist file
+
+When producing a framework, the Kotlin/Native compiler generates the information property list file, `Info.plist`.
+You can customize its properties with the corresponding binary option:
+
+| Property                     | Binary option              |
+|------------------------------|----------------------------|
+| `CFBundleIdentifier`         | `bundleId`                 |
+| `CFBundleShortVersionString` | `bundleShortVersionString` |
+| `CFBundleVersion`            | `bundleVersion`            |
+
+For that, pass the `Xbinary=$option=$value` compiler flag or set the `binaryOption(option, value)` Gradle DSL for
```

Comment:
I found such "for that" too synthetic.
Could we change it to the full sentence, like "To customize ..., pass the" or "To enable the feature ...". etc.